downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc

```diff
@@ -0,0 +1,45 @@
+
+[id="proc-deploy-eda-controller-with-aap-operator-ocp_{context}"]
+
+
+
+.Prerequisites
+
+* You have installed {OperatorPlatform} on {OCPShort}.
+* You have installed and configured {ControllerName}.
+
+.Procedure
+
+. Navigate to menu:Operators[Installed Operators].
+
+. Create a new Event-Driven Ansible custom resource. 
++
+This takes you to the Form View to customize your installation.
+
+. Specify your controller URL. 
++
+If you deployed automation controller in Openshift as well, you can find the URL on the Routes page.
++
+[NOTE]
+====
+This is the only required customization, but you can customize other options using the UI form or directly in the YAML configuration tab, if desired.
+====
+
+. Click btn:[Create].
+This deploys {EDAcontroller} in the namespace you specified. After a couple minutes when the installation is marked as *Successful*, you can reach the UI. 
```

Comment:
I'm confused by the phrase, "you can reach the UI".  Are they taken outside of the OpenShift UI to some other UI?

Reply:
I can add a little more context here.  When you deploy EDA using the AAP Operator on Openshift, the following is happening behind the scenes:
* containers are started inside the Openshift cluster which run the services needed to run the EDA application
* The service that provides the User Interface for EDA is not accessible via the browser until it is "exposed" publicly using Service and Route resources (which are automatically set up  by the operator when deploying EDA.
* Openshift Routes do some DNS magic that creates a URL for the EDA UI that is publically available.  So once the containers are all running and the route is created (all handled by the operator), the user should be able to access/reach the URL in their browser. 

tl;dr, when the deployment is complete, the URL for the EDA UI (found on the Route page in the Openshift UI), will be available/reacheable/accessible in the user navigates to it in their browser.